metadata.json

```diff
@@ -29,14 +29,17 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": "3.x"
-    },
-    {
-      "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.4.0"
```

Comment:
Plus it looks like it might have gotten a bit heavy handed and removed the puppet requirement.